### ISSUE TITLE
Read receipts jump with lazy loading of members enabled

### ISSUE THREAD

**Issue by bwindels** [vector-im/riot-web#7260]
RR positions only seem to update once the bg /members update completes, which causes a nasty pop as both the RRs suddenly click into position and the /members list loads.  Instead we should surely be showing the RRs in the right place but just populating up their profiles (if needed) once /members loads?
    * I just saw this again, and this time the RRs suddenly changed position quite long after the memberlist had LL'd (and updated from a handful to the full list).  So i don't think it's blocked on just the bg /members completing, but something else.

**Comment by bwindels** [comment]
The RR issue really looks the same as #7261 , it's just the catchup sync taking a long time. Investigated a bit but couldn't reproduce them being blocked on /members.

I'd leave it as it is and hope for /sync to get faster?

**Comment by bwindels** [comment]
Split of from #7182

**Comment by ara4n** [comment]
yeah, i'm hoping that the spinner behaviour will make the RR jump less surprising (and/or the faster /sync will avoid the RRs going slowly). Let's close this once both have landed unless it's still a problem.

**Comment by turt2live** [comment]
I have seen this. The read receipts in general for a room appear to be blocked on the first load of the member list.

**Comment by turt2live** [comment]
On average it's taking more than a few seconds to load members for me, so I'd appreciate anything that makes the popin effect less dramatic.

**Comment by erikjohnston** [comment]
Yeah, its quite annoying when you first load up a large room. I failed to find this, so created a duplicate issue (#7449):

> Lost of the little avatar bubbles for read receipts are blank until the member list is fetched. This looks a bit weird when you have a line of read receipts and they're full of gaps.
>
> It might be nice to have some visual place holder for those avatars, or possibly not event display any read receipts until the members are loaded (to stop the slightly weird experience of only seeing a subset of receipts at first)

**Comment by anoadragon453** [comment]
I'm still seeing this today:

![image](https://user-images.githubusercontent.com/1342360/59928501-cffb2b80-9436-11e9-9d31-2a4e63d98ccb.png)

**Comment by bwindels** [comment]
from your screenshot, looks more like some avatars couldn't be loaded?
